src/app/dev/DevToys.Api/Tool/GUI/Components/IUIDropDownListItem.cs

```diff
@@ -28,6 +28,35 @@ internal UIDropDownListItem(string? text, object? value)
     public string? Text { get; }
 
     public object? Value { get; }
+
+    public static bool operator ==(UIDropDownListItem item1, UIDropDownListItem item2)
+        => Equals(item1, item2);
+
+    public static bool operator !=(UIDropDownListItem item1, UIDropDownListItem item2)
+        => !Equals(item1, item2);
+
+    public override bool Equals(object? other)
+        => Equals(other as UIDropDownListItem);
+
+    public bool Equals(UIDropDownListItem? other)
+    {
+        if (other is null)
+        {
+            return false;
+        }
+
```

Comment:
How about adding:
```suggestion
        if (ReferenceEquals(this, other))
        {
            return true;
        }
```
It basically check if these two objects are the exact same instance. It can help making the comparison slighlty faster if `Value` is a complex object to compare or if `Text` is a long string.